department-of-veterans-affairs/va.gov-cms
# VACMS-1390 facility status migration

## Description

See #1390. 

## Testing done


## Screenshots


## QA steps

Pull locally, or ssh into devshop
1. Check the status of the migration 
`drush migrate-status va_node_health_care_local_facility_status`  
- [x] Validate that it returns (after several minute wait) :
```
Group: facility_status (facility_status)   Status  Total  Imported  Unprocessed  Last imported       
 va_node_health_care_local_facility_status  Idle    1009   0         1009            
```
The exact number may fluctuate slightly depending on the source asp paages.

2. Run the migration as an update migration 
`drush migrate-import va_node_health_care_local_facility_status --update`

3. The migration may take a while, but after it is completed check the status:
`drush migrate-status va_node_health_care_local_facility_status`  

- [x]  Validate that the numbers look something like:
```
Group: facility_status (facility_status)   Status  Total  Imported  Unprocessed  Last imported       
 va_node_health_care_local_facility_status  Idle    1009   1006      1            2020-04-13 22:03:45
```
The numbers may deviate slightly.

4. Visit `/locations/fort-knox-va-clinic` 
- [x] Validate that it has an operating status (most likely closed) and a status info message and that the page is unpublished. 

**Operating Status ✅ | Operating Status - more info ⚠️**
<img width="609" alt="Screen Shot 2020-04-14 at 8 26 28 AM" src="https://user-images.githubusercontent.com/16477724/79236301-04a6fc00-7e2a-11ea-89dc-19d43081e052.png">


- [x] Validate that the last updated date reflects when you ran the migration.

5. Visit `/locations/wheatland-va-mobile-clinic` 
- [x] Validate that it has an operating status (most likely closed) and a status info message and that the page is unpublished.

**Operating Status ✅ | Operating Status - more info ⚠️**
<img width="612" alt="Screen Shot 2020-04-14 at 8 26 10 AM" src="https://user-images.githubusercontent.com/16477724/79236347-125c8180-7e2a-11ea-8ed0-e2193f4a0197.png">

- [x] Validate that the last updated date reflects when you ran the migration.
